README.md

```diff
@@ -891,6 +891,7 @@ This includes libraries for things like XML, JSON, CSV, and other similar format
 * [theft][302] - Property-based testing (similar to [Quickcheck][301]). [``MIT``][MIT]
 * [Unity][296] - Simple unit testing framework. [``MIT``][MIT]
 * [utest][386] - Single-header unit testing library. [``Unlicense``][Unlicense]
+* [Nala][560] - A test framework for C projects. [``MIT``][MIT]
```

Comment:
Can you put this alphabetically in the list please?

Reply:
Sure, it should be fine now.